tests/python/utils.py

```diff
@@ -417,6 +418,35 @@ def inner_fn(*args, **kwargs):
     return inner_fn
 
 
+UPDATED_SDPA = LooseVersion(torch.__version__) >= LooseVersion("2.7.0")
+
+
+def get_sdpa_rng_nvf_tensors(fd: FusionDefinition):
+    dtype = DataType.UInt64 if UPDATED_SDPA else DataType.Int
+    is_cpu = False if UPDATED_SDPA else True
+    philox_shape = [-1] if UPDATED_SDPA else []
+    philox_seed = fd.define_tensor(
+        shape=philox_shape,
+        dtype=dtype,
+        is_cpu=is_cpu,
+    )
+    philox_offset = fd.define_tensor(
+        shape=[],
+        dtype=dtype,
+        is_cpu=is_cpu,
+    )
+    return philox_seed, philox_offset
+
+
+def get_sdpa_rng_tensors():
```

Comment:
```suggestion
def create_sdpa_rng_state() -> tuple[torch.Tensor, torch.Tensor]:
```
Or `create_sdpa_rng_state_tensors`

Reply:
"get" is unclear whether the tensors were created or reused.